openedx/core/djangoapps/zendesk_proxy/utils.py

```diff
@@ -94,7 +94,17 @@ def create_zendesk_ticket(
                 )
             )
         if additional_info:
-            ticket = json.loads(response.text)['ticket']
+            try:
+                ticket = response.json()['ticket']
+            except ValueError as e:
```

Comment:
Nit: It's possible for this to throw a `KeyError` as well, if the JSON parses but it doesn't have the `ticket` key.